src/agent/advertising_proxy.cpp

```diff
@@ -0,0 +1,322 @@
+/*
+ *    Copyright (c) 2020, The OpenThread Authors.
```

Comment:
```suggestion
 *    Copyright (c) 2021, The OpenThread Authors.
```

Reply:
Actually, those files (and this PR) are created at the end of 2020 :)